plugins/techdocs/api-report.md

```diff
@@ -118,11 +118,11 @@ export const DocsTable: ({
   actions?:
     | (
         | Action<DocsTableRow>
+        | ((rowData: DocsTableRow) => Action<DocsTableRow>)
```

Comment:
You'll want to just drop this change from the PR, should be fine

Reply:
Thanks! I didn't notice that I had included this change in my commits!